code/src/index.css

```diff
@@ -1,13 +1,188 @@
   -moz-osx-font-smoothing: grayscale;
+  background-color: black;
+  color: whitesmoke;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+/* NAVBAR */
+.nav-bar {
+  margin: 0px 30px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav-bar button {
+  text-align: center;
+  height: 50px;
+  width: 150px;
+  margin-top: 100px;
+  margin-right: 100px;
+  border-radius: 50px;
+  font-size: 20px;
+  background: red;
+  color: white;
+  border: 0;
+}
+.nav-bar button:hover {
+  border: 2px solid white;
+  font-size: 22px;
+}
+.nav-bar h1 {
+  margin-left: 90px;
+  font-size: 70px;
+}
+.nav-bar h1 img {
+  height: 100px;
+}
+/* MOVIE LIST */
+
+.movie-list-card {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  word-wrap: break-word;
+  justify-content: center;
+  align-content: baseline;
+}
+
+/* MOVIE CARD */
+
+.movie-page-background {
+  height: 100%;
+}
+
+.overlay-container {
+  position: relative;
+}
+
+.text-overlay {
+  position: absolute;
+  bottom: 0;
+  color: #fff;
+  background: rgba(0, 0, 0, 0.7);
+  width: 100%;
+  height: 100%;
+  transition: 0.3s ease;
+  opacity: 0;
+}
+
+.text-overlay h1 {
+  position: absolute;
+  bottom: 50px;
+  margin-left: 20px;
+  font-size: 20px;
+}
+
+.text-overlay p {
+  position: absolute;
+  bottom: 0;
+  padding: 20px;
+  font-size: 15px;
+}
+
+.text-overlay:hover {
+  opacity: 1;
+}
+
+.text-overlay p,
+.text-overlay h1 {
+  color: whitesmoke;
+  text-decoration: none;
+}
+
+/* MOVIE-PAGE */
+
+.movie-page-content {
+  transform: translate(0px, 200px);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.movie-page-text-content {
+  width: 342px;
+  margin-bottom: 50px;
+}
+
+.movie-page-poster {
+  border: 2px solid whitesmoke;
+}
+
+.star {
+  margin-right: 10px;
+}
+/* BACK ICON */
+
+.button {
+  position: absolute;
+  top: 0;
+  left: 0;
+  margin: 50px;
+  display: flex;
+  justify-content: center;
+  background: rgba(0, 0, 0, 0.55);
+  padding-right: 5px;
+  border-radius: 5px;
+  /* border: 1px solid green; */
+}
+
+.circle {
+  width: 35px;
+  height: 20px;
+  position: relative;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  display: flex;
+  justify-content: center;
+  padding: 9px 0 10px 8px;
+  border: 1px solid #fff;
+  border-radius: 50%;
+  background-color: #fff;
+  margin: 5px;
+}
+
+.arrow {
+  position: absolute;
+  width: 16px;
+  height: 16px;
+  border-top: 5px solid rgba(0, 0, 0, 0.55);
+  border-left: 5px solid rgba(0, 0, 0, 0.55);
+  transform: rotate(-45deg);
+}
+
+.button-p {
+  font-size: 17px;
+  align-self: center;
+  color: #fff;
+  font-weight: 700;
+  margin: 0 2px;
+}
+
+.button-p:hover {
+  margin-left: 7px;
+  transition: 0.2s ease-in-out;
+}
+
+@media (min-width: 900px) {
+  .movie-page-content {
+    flex-direction: row;
+    justify-content: center;
+  }
+
+  .movie-page-text-content {
+    margin: 50px;
+  }
+}
```

Comment:
Great that you have a media query for the desktop view. It works well on tablet but I miss a mobile view. 📵